SW_Model.c

```diff
@@ -90,7 +95,7 @@ void SW_MDL_construct(void) {
 
 #ifndef STEPWAT
 	/* already set by user-provided seed in steppe */
-	RandSeed(0);
+	RandSeed(0,&markov_rng);

```

Comment:
I don't understand why you put all of this code related to `markov_rng` into the module `SW_Model.c`; it is not used by this code. Why not put it instead in `SW_Markov.c`, define `markov_rng` in that module, and set the seed by the function `SW_MKV_construct()`?